src/script_lua.c

```diff
@@ -945,19 +908,10 @@ static int luaRedisStatusReplyCommand(lua_State *lua) {
  * Set the propagation of write commands executed in the context of the
  * script to on/off for AOF and slaves. */
 static int luaRedisSetReplCommand(lua_State *lua) {
-    int argc = lua_gettop(lua);
```

Comment:
Why removing this? You still need to check that you are getting an argument no?

Reply:
mistake  fixed: